CHANGELOG.md

```diff
@@ -1,5 +1,24 @@
 # Changelog
 
+## 17.1.0 (12/18/24)
```

Comment:
We should first include a section with new features in this minor release: https://goteleport.com/docs/upcoming-releases/#1710.